src/ducks/SANCharts/ChartPage.js

```diff
@@ -48,18 +46,12 @@ const LoadableChartMetricsTool = Loadable({
 
 const getChartInitialState = props => {
   let passedState
-
   if (props.location && props.location.search) {
-    const parsedQuery = mapParsedTrueFalseFields(
```

Comment:
Why was removed? :)
Sometimes get string 'true' of string 'false' which is always boolean true in any comparisions. (for example variable 'isAdvancedView' from query)

Reply:
Don't worry, just a merge conflicts resolve side effect. Will be returned in the next commit